### ISSUE TITLE
Change URI of Person Mentioned in Title

### ISSUE THREAD

**Issue by davidamichelson** [srophe/srophe-app-data#946]
@wlpotter 
This URI: https://github.com/srophe/srophe-app-data/blob/3b130b96fe1ce37b295a2dcd8aff0cae758cf4d1/data/manuscripts/tei/1120.xml#L505

Should be changed to http://syriaca.org/person/511 (Gregory Nazianzen).

Can I just make this change on this Dev branch?

Thanks,
DM

**Comment by davidamichelson** [comment]
Actually, I am not sure on this, I need to check it and I realize it would mean overruling Wright which we are not prepared to do yet.

**Comment by wlpotter** [comment]
@davidamichelson we can wait on this; but when you want to make a change you can edit the record on dev directly.

by the way, I will make a separate issue for moving mss records to master.
